.github/workflows/lint.yml

```diff
@@ -62,4 +62,4 @@ jobs:
         run:            cargo install cargo-spellcheck
 
       - name:           Run spellcheck
-        run:            cargo spellcheck check -m 1 -vv $(find modules/currency-exchange/src -name "*.rs")
+        run:            cargo spellcheck check -m 1 -vv $(find modules/dispatch/src -name "*.rs") $(find modules/currency-exchange/src -name "*.rs")
```

Comment:
I think we should be fixing typos for `modules/*` instead of doing each one by one. We can then add `bin/*`, then `relays/*`, etc. in order to keep the typos PRs relatively manageable in size

Reply:
Roger, will do that in the way.